test/testutils.ts

```diff
@@ -172,6 +172,13 @@ export function decodeRevertReason (data: string, nullIfNoMatch = true): string
   } else if (methodSig === '0x4e487b71') {
```

Comment:
I think this entire method is obsolete... I can see it decodes an obsolete signature of FailedOp (we removed the "paymaster" parameter a long time ago

Reply:
That's right, this method is not really maintained. What should we do about it?